README.md

```diff
@@ -91,6 +91,7 @@ Applications with support of IPTV streams.
 - [APTV](https://apps.apple.com/us/app/aptv/id1630403500) - A high-quality live streaming source player app that can preview in real time. It is a multifunctional player that can play and view (requires support from the live streaming source).
 - [TV Set](https://apps.apple.com/de/app/tv-set/id1660197316) - Free IPTV app for watching tv on iOS, tvOS, and macOS.
 - [Live TV Player](https://apps.apple.com/us/app/live-tv-player/id6448868009) - Free IPTV Player by TechArt Apps, supports .m3u playlists, Picture-in-Picture, AirPlay.
+- [https://github.com/mfat/tvhplayer] - Desktop client for TVheadend using vlc as playback engine. Features include EPG support, remote (DVR) and local recording. It works on Linux, macOS and Windows.
```

Comment:
- missing app name
- wrong section
- the description should consist of one sentence

https://github.com/iptv-org/awesome-iptv/blob/master/.github/CONTRIBUTING.md

Reply:
Just fixed the name and description but not sure what is the proper section for this. It works on linux, mac and windows.

Reply:
You can add a link to each of the corresponding sections. But this section is for iOS apps, so it's definitely the wrong place for it.